pkg/discovery/dtofactory/node_entity_dto_builder.go

```diff
@@ -163,6 +163,7 @@ func (builder *nodeEntityDTOBuilder) BuildEntityDTOs(nodes []*api.Node) []*proto
 			continue
 		}
 
+		entityDto.ProviderPolicy = &proto.EntityDTO_ProviderPolicy{AvailableForPlacement: &isAvailableForPlacement}
```

Comment:
Please print a warning log if the isAvailableForPlacement is false to help troubleshooting

Reply:
Done!